packages/astro/src/core/build/plugins/plugin-internals.ts

```diff
@@ -1,19 +1,23 @@
 import type { Plugin as VitePlugin } from 'vite';
 import type { BuildInternals } from '../internal.js';
-import type { AstroBuildPlugin } from '../plugin.js';
 import type { StaticBuildOptions } from '../types.js';
 import { normalizeEntryId } from './plugin-component-entry.js';
 
-function vitePluginInternals(
-	input: Set<string>,
-	opts: StaticBuildOptions,
+export function pluginInternals(
+	options: StaticBuildOptions,
 	internals: BuildInternals,
 ): VitePlugin {
+	let input: Set<string>;
```

Comment:
This variable is never used, so we can remove it. Like, we assign values in `configResolved`, but we never read those values anywhere